diff --git a/Makefile b/Makefile
index 2d31fee..23f65bc 100644
--- a/Makefile
+++ b/Makefile
@@ -659,6 +659,15 @@ else
    @$(LINKER) $(SOFLAGS) -o $(LIBBLIS_SO_OUTPUT_NAME) $? $(LDFLAGS)
 endif
 endif
+	$(LINKER) $(SOFLAGS) -o $(BASE_LIB_PATH)/$(DEB_LIBBLAS) $(LDFLAGS) \
+       $(filter-out $(BASE_OBJ_CBLAS_PATH)/%.o,
+           $(filter $(BASE_OBJ_BLAS_PATH)/%.o, $(MK_BLIS_OBJS))) \
+       -Wl,--soname,$(DEB_LIBBLAS) \
+       -lblis -L.
+	$(LINKER) $(SOFLAGS) -o $(BASE_LIB_PATH)/$(DEB_LIBCBLAS) $(LDFLAGS) \
+       $(filter $(BASE_OBJ_CBLAS_PATH)/%.o, $(MK_BLIS_OBJS))) \
+       -Wl,--soname,$(DEB_LIBCBLAS) \
+       -lblis -L.
 
 # Local symlink for shared library.
 # NOTE: We use a '.loc' suffix to avoid filename collisions in case this
