diff --git a/Makefile b/Makefile
index 9cc81c9..0b1c0d6 100644
--- a/Makefile
+++ b/Makefile
@@ -658,6 +658,15 @@ else
    @$(LINKER) $(SOFLAGS) -o $(LIBBLIS_SO_OUTPUT_NAME) $? $(LDFLAGS)
 endif
 endif
+	$(LINKER) $(SOFLAGS) -o $(BASE_LIB_PATH)/$(DEB_LIBBLAS) $(LDFLAGS) \
+       $(filter-out $(BASE_OBJ_CBLAS_PATH)/%.o, $(filter $(BASE_OBJ_BLAS_PATH)/%.o, $(MK_BLIS_OBJS))) \
+       -Wl,--soname,$(DEB_LIBBLAS) -Wl,--version-script=$(LDS_BLAS) \
+       -lblis -L.
+	$(LINKER) $(SOFLAGS) -o $(BASE_LIB_PATH)/$(DEB_LIBCBLAS) $(LDFLAGS) \
+       $(filter $(BASE_OBJ_CBLAS_PATH)/%.o, $(MK_BLIS_OBJS)) \
+       -Wl,--soname,$(DEB_LIBCBLAS) -Wl,--version-script=$(LDS_CBLAS) \
+       -lblis -L.
+

 # Local symlink for shared library.
 # NOTE: We use a '.loc' suffix to avoid filename collisions in case this
