diff -ru lapack-3.8.0.orig/CMakeLists.txt lapack-3.8.0/CMakeLists.txt
--- lapack-3.8.0.orig/CMakeLists.txt    2017-11-13 04:15:54.000000000 -0000
+++ lapack-3.8.0/CMakeLists.txt 2019-07-11 15:41:42.535524400 -0000
@@ -38,14 +38,8 @@
 # in the CMake Wiki:
 #  http://www.cmake.org/Wiki/CMake_RPATH_handling#Mac_OS_X_and_the_RPATH
 set(CMAKE_MACOSX_RPATH ON)
-set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(CMAKE_SKIP_BUILD_RPATH TRUE)
 set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
-list(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES ${CMAKE_INSTALL_FULL_LIBDIR} isSystemDir)
-if("${isSystemDir}" STREQUAL "-1")
-  set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR})
-  set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-endif()
-
 
 # Configure the warning and code coverage suppression file
 configure_file(
